mprocs.yml

```diff
@@ -19,6 +19,18 @@ procs:
       PORT: '3001'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18184
       BROWSER: none
+  guardian-ui-3:
+    shell: yarn dev:guardian-ui
+    env:
+      PORT: '3003'
```

Comment:
nit: maybe we need to assign contiguous ports to the guardian UIs thus push gateway UI port back to 3004 or 3005?

Reply:
Good call. Was lazy. 😅